views/partials/header.ejs

```diff
@@ -48,4 +48,71 @@
 
 	</nav>
 
+	<nav class="desktop-menu">
+		<section class="top-bar">
+		</section>
+		<section class="first-row">
+			<div class="first-col">
+				Live date here..
+			</div>
+			<div class="second-col">
+				<picture>
```

Comment:
It is always good to have an element in the header that refreshes the page or leads the user back to the home page.  since users are used to having this feature with the logo i would recommend to do the same.

Reply:
you can do this by wrapping the logo in a link. and the link in a div